packages/fxa-settings/src/components/CardHeader/index.tsx

```diff
@@ -115,11 +115,12 @@ function isBasicWithCustomSubheading(
 const serviceLogos: {
   [key in MozServices]?: ReactElement;
 } = {
+  // This is not inlined because text inside of an SVG can have rendering problems
```

Comment:
Q: Are there other places we are inlining SVGs that might have similar problems?

I tried this <kbd>grep</kbd> and it looks like we potentially inline .svg's in about 130 other places:

```sh
git grep "ReactComponent as " packages/**/*.tsx | wc -l # 131
```

&hellip; or is the rendering issues specifically because of the text in the SVG?
(apologies, the SVG in question uses `<path>` and `<g>`, so pretty hard to `grep` other SVGs for potential similar issues).

Reply:
I haven't seen this happen for anything else except the Pocket logo and the Mozilla Monitor logo. It's best practice to not include text inside SVGs but there's exceptions for branding and the accessibility folks were fine with the Monitor logo in text too. I saw this happen with 3 different Monitor SVGs (before and after SVGO), but yeah - it's probably some path issue, like, there's some delta that's probably causing this, it's just weird that the only two times I've seen it were specifically text in the SVG and I haven't investigated any further.